windows-observ-lib/utils.libsonnet

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I think most of these are pretty common utils. I don't recall. Did we intend to merge these with grafonnet or with the common-lib?

Reply:
Thnks, Moved to commonlib